Sources/SwiftSDKGenerator/Generator/SwiftSDKGenerator+Copy.swift

```diff
@@ -47,29 +47,37 @@ extension SwiftSDKGenerator {
           )
         }
 
-        let sdkUsrLib64Path = sdkUsrPath.appending("lib64")
-        try await generator.copyFromDockerContainer(
-          id: containerID,
-          from: FilePath("/usr/lib64"),
-          to: sdkUsrLib64Path
-        )
-        try await createSymlink(at: pathsConfiguration.sdkDirPath.appending("lib64"), pointingTo: "./usr/lib64")
+        if case let containerLib64 = FilePath("/usr/lib64"),
+           try await generator.doesPathExist(containerLib64, inContainer: containerID) {
```

Comment:
It already _is_ called with `failIfNotExists: false`... See Line 57.
Not sure where the second check is coming from...

Reply:
> It already _is_ called with `failIfNotExists: false`... See Line 57. Not sure where the second check is coming from...
> Ah, it actually needs to be failIfNotExists: true now to avoid the second check...

Oops, I got it the wrong way around in my comment - I should have said `failIfNotExists: true`. 🙃